windows/win_zip.ps1

```diff
@@ -0,0 +1,241 @@
+#!powershell
+# This file is part of Ansible
+#
+# Copyright 2015, Phil Schwartz <schwartzmx@gmail.com>
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# WANT_JSON
+# POWERSHELL_COMMON
+
+$params = Parse-Args $args;
+
+$result = New-Object psobject @{
+    win_zip = New-Object psobject
+    changed = $false
+}
+
+# Check creates to begin
+If ($params.creates) {
+    If (Test-Path $params.creates) {
+        Exit-Json $result "The 'creates' file already exists."
+    }
+}
+
+# Global flags
+$isLeaf = $false
+$isContainer = $false
+
+# Check if PSCX is installed
+$list = Get-Module -ListAvailable
+If (-Not ($list -match "PSCX")) {
+    Set-Attr $result.win_zip "pscx_status" "absent"
+    $pscxPresent = $false
+}
+Else {
+    $pscxPresent = $true
+    Set-Attr $result.win_zip "pscx_status" "present"
+}
+
+# Import
+Try {
+    If ($pscxPresent) {
+        Try {
+            Import-Module 'C:\Program Files (x86)\Powershell Community Extensions\pscx3\pscx\pscx.psd1'
```

Comment:
Yep good point @trondhindenes,  the reason that was like this is because when I originally wrote this I was auto-installing PSCX and the module couldn't import it as it was running in the same PS session, so I had to import it from the install location.  Just a bad lingering snippet.  Will fix.